.claude/hooks/pre_tool_use.py

```
@@ -7,196 +7,231 @@
-        with open(log_path, 'w') as f:
+        with open(log_path, "w") as f:
             json.dump(log_data, f, indent=2)
-        
+
         sys.exit(0)
-        
+
     except json.JSONDecodeError:
         # Gracefully handle JSON decode errors
         sys.exit(0)
     except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)